app/src/main/java/org/mozilla/focus/download/GetDownloadFileHeaderTask.java

```diff
@@ -32,16 +36,22 @@ protected HeaderInfo doInBackground(String... params) {
             connection = (HttpURLConnection) new URL(params[0]).openConnection();
             connection.setRequestMethod("HEAD");
             String headerField = connection.getHeaderField("Accept-Ranges");
+            String strContentLength = connection.getHeaderField("Content-Length");
             if (headerField != null && headerField.equals("bytes")) {
                 headerInfo.isSupportRange = true;
             }
+            if (strContentLength != null) {
+                headerInfo.contentLength = Long.parseLong(strContentLength);
+            }
             connection.getResponseCode();
             connection.disconnect();
 
         } catch (SSLHandshakeException e) {
             headerInfo.isValidSSL = false;
         } catch (IOException e) {
-            e.printStackTrace();
+            Log.e(TAG, "IOException");
```

Comment:
Why not keep `e.printStackTrace()` to show detail call stack for debugging?

Reply:
fixed